nvi-commons/src/main/java/no/sikt/nva/nvi/common/db/dto/CandidateDb.java

```diff
@@ -0,0 +1,234 @@
+package no.sikt.nva.nvi.common.db.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import java.util.List;
+import no.sikt.nva.nvi.common.db.Typed;
+import no.sikt.nva.nvi.common.db.WithCopy;
+import no.sikt.nva.nvi.common.db.dto.CandidateDb.Builder;
+
+public class CandidateDb implements WithCopy<Builder>, Typed {
```

Comment:
Do we need a class here? It seems like all the fields got same naming as variables, then we can make this class to record with builder and copy() method. But it's up to you :)